pom.xml

```diff
@@ -116,7 +116,7 @@
             <dependency>
                 <groupId>org.postgresql</groupId>
                 <artifactId>postgresql</artifactId>
-                <version>42.3.3</version>
+                <version>42.5.0</version>
```

Comment:
Could you merge the master branch into this branch so this does not show up as a change made by this PR?

Reply:
Merged